misc/docs-linter/main.go

```diff
@@ -72,21 +73,29 @@ func execLint(cfg *cfg, ctx context.Context) error {
 			return err
 		}
 
+		// Execute JSX extractor
 		fileJSXMap[filePath] = extractJSX(fileContents)
 
 		// Execute URL extractor
 		fileUrlMap[filePath] = extractUrls(fileContents)
+
+		// Execute local link extractor
+		fileLocalLinkMap[filePath] = extractLocalLinks(fileContents)
 	}
 
 	// Run linters in parallel
 	g, _ := errgroup.WithContext(ctx)
 
 	g.Go(func() error {
-		return lintJSX(fileJSXMap, ctx)
+		return lintJSX(fileJSXMap)
+	})
+
+	g.Go(func() error {
+		return lintURLs(fileUrlMap, ctx)
```

Comment:
Was it a bug that this was previously missing?

Reply:
No, the `lintURLs` functionality was there before, I just made it a bit confusing with the git diff. For the terminology in the linter, before this PR, URLs == links, now, URLs == URLs, local file links == links. Might be a bit confusing, but I couldn't come up with a more practical naming scheme. I'm open to suggestions